lib/car.rb

```diff
@@ -0,0 +1,16 @@
+require 'pry'
+
+class Car
+  attr_reader :car, :monthly_payment, :loan_length, :model, :make
+
+  def initialize(car, monthly_payment, loan_length)
+    @car = car
+    @monthly_payment =  monthly_payment
+    @loan_length = loan_length
+    @make, @model = @car.split(" ")
+  end
+
+  def total_cost
+    @monthly_payment * loan_length
+  end
+end
```

Comment:
The only missing parts to this class is the `@color` attribute and the `paint` method. Did you miss those in the interaction pattern for Iteration 1?
```
pry(main)> car.color
#=> nil

pry(main)> car.paint!(:blue)

pry(main)> car.color
#=> :blue
```

Reply:
Yes. I completely missed that section! I'll have to slow down and pay attention next time!